### PR TITLE
Added tests around handleChangeInUserRequestToAlexa method.

### PR DESCRIPTION
**Pull request by s-maheshbabu**
Added tests around handleChangeInUserRequestToAlexa method in ChatWindow component.

### CODE REVIEW

File: src/ChatWindow.js [hunk 1]
```diff
@@ -71,6 +71,7 @@ class ChatWindow extends Component {
   Update the state as the user is typing into the input box
   */
   handleChangeInUserRequestToAlexa(event) {
+    event.preventDefault();
```
Comment (gootla-manasa) [inline src/ChatWindow.js]: Follow the same naming convention for event through out the code i.e., either 'e' or 'event'.

File: src/ChatWindow.test.js [hunk 2]
```diff
@@ -130,3 +131,25 @@ it("handles gracefully when the input form is submitted with a null or empty req
   // Nothing about the state should have changed.
   expect(finalState).toEqual(originalState);
```
Comment (gootla-manasa) [inline src/ChatWindow.test.js]: Verify that preventDefault() is being called.

Comment (gootla-manasa) [reply src/ChatWindow.test.js]: Verify preventDefault is being called in other tests as well wherever necessary.

Also, could you make preventDefaultSpy a class variable instead?

File: src/ChatWindow.js (hunk 1 above)
Comment (s-maheshbabu) [reply src/ChatWindow.js]: Done. Renamed to event.

File: src/ChatWindow.test.js (hunk 2 above)
Comment (s-maheshbabu) [reply src/ChatWindow.test.js]: Done.

File: src/ChatWindow.test.js [hunk 3]
```diff
@@ -139,7 +143,7 @@ it("handles the user's input as they are typing their request (before submission
   });
 
   const expectedUserRequestToAlexa = "mock request";
-  const preventDefaultSpy = jest.fn();
+  preventDefaultSpy = jest.fn();
```
Comment (gootla-manasa) [inline src/ChatWindow.test.js]: This is not needed. You can use the instance variable.

Comment (s-maheshbabu) [reply src/ChatWindow.test.js]: Done.

**Pull request merged**